src/main/java/co/kr/muldum/domain/token/repository/TokenRepository.java

```diff
@@ -3,6 +3,7 @@
 import co.kr.muldum.domain.token.model.RefreshToken;
 import org.springframework.data.jpa.repository.JpaRepository;
 
-public  interface TokenRepository extends JpaRepository<RefreshToken, String> {
+public interface TokenRepository extends JpaRepository<RefreshToken, String> {
 
+    boolean deleteByToken(String token);
```

Comment:
반환타입을 Optional 로 감싸고 
```suggestion
    Optional<void> deleteByToken(String token);
```
로 감싸주시고 서비스 로직에서 예외를 발생시키는 게 어떨까요?